app/src/main/java/ch/epfl/polybazaar/database/DataSnapshot.java

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.polybazaar.database;
+
+
+
+import java.util.Map;
+
+public class DataSnapshot {
+
```

Comment:
Please mind spacing between methods/variable definitions.

Reply:
Also, what's a DataSnapshot? Is it a DocumentSnapshot, is it a DocumentReference, a QuerySnapshot? The name doesn't tell much about the object. Maybe call it DocumentSnapshot, i.e. overload DocumentSnapshot?